docs/azure_arc_jumpstart/azure_edge_iot_ops/aio_manufacturing/_index.md

```diff
@@ -0,0 +1,437 @@
+
+  ![Screenshot showing how to navigate to ADX dashboard reports](./61.png)
+
+- Select "Import dashboard from file" to select previously copied file from the Client VM to your local machine or the *C:\AIO\adx_dashboard* folder on the Client VM.
+
+  ![Screenshot showing the import dashboard file](./62.png)
+
+- Choose to import the *dashboard.json* file.
+
+  ![Screenshot showing the dashboard report JSON file to import](./63.png)
+
+- Confirm the dashboard report name, accept the suggested name (or choose your own), and click Create.
+
+  ![Screenshot showing the dashboard report name confirmation](./64.png)
+
+- It might take a couple of minutes for the dashboard to display some data in the report after deployment. Click Save to save the dashboard report in ADX.
+
+  ![Screenshot showing the empty data in orders dashboard report](./65.png)
+
+- Contoso staff can start levergaing the dashboards, for instance, a regional manager overseeing three plants might use the ADX dashboard to assess the collective OEE of these facilities. The dashboard aggregates essential metrics such as Overall OEE, Performance, and Availability, offering a consolidated view of productivity and efficiency. It highlights the most prevalent current shift's waste reason—in this case, 'Overcooked' and quantifies waste volumes alongside unplanned downtimes and their duration in minutes.
+
+- Furthermore, these stakeholders often engage in comparative analysis. The ADX dashboard enables them to compare the performance across the different shifts operating within a single plant, discerning the OEE for primary products like donuts, bread, and pastries. It also facilitates a comparative review of the OEE across the three plants located in Seattle, Mexico City, and Miami. Quality indicators are also on display, ensuring products meet the required specifications in size and weight.
+
+  ![Screenshot showing the OEE ADX dashboard](./66.png)
+
+
+- Critical quality-affecting factors, such as the oil temperature in fryers which can influence the color, size, or doneness of donuts, are also monitored. This level of detail provides executives with the data needed to make informed decisions, ensuring product consistency and operational excellence across the board.
+
+## Cleanup deployment
+
+- To clean up your deployment, simply delete the resource group using Azure CLI or Azure portal.
+
+  ```shell
+  az group delete -n <name of your resource group>
+  ```
+
+  ![Screenshot showing az group delete](./67.png)
+
+  ![Screenshot showing group delete from Azure portal](./68.png)
+
+- If you used Azure Developer CLI to deploy then ```azd down``` can be used instead.
+
+  ![Screenshot showing azd down](./69.png)
+
+## Next steps
+
+If you still having issues with the deployment, please refer to the [Troubleshooting](../troubleshooting//) section. Otherwise, if you have additional questions or feedback, please refer to the [FAQ](../../../faq/) section.
```

Comment:
See updated Agora docs for these relative paths. Also, where is the troubleshooting section?